js-miniapp-sample/src/pages/user-details.js

```diff
@@ -157,13 +157,15 @@ export const dataFetchReducer = (state: State, action: Action) => {
         isLoading: true,
         isError: false,
         hasRequestedPermissions: false,
+        hasRequestedPointPermissions: false,
```

Comment:
I think it's still incorrect to set `hasRequestedPointPermissions` under `FETCH_INIT`, `FETCH_SUCCESS`, `FETCH_FAILURE` because these are for fetching the other user details, not the points details.

Reply:
@corycaywood Initially, I thought only updating to `isPointsError` should enough to fix the issue.
But, I have rechecked that we also need to set `hasRequestedPointPermissions: false` in here to fix the issue.

Reply:
@khairul-alam-licon What issue does it fix to have `hasRequestedPointPermissions` here? I can see that you need to set `hasRequestedPointPermissions: true` under the `POINTS_FETCH_SUCCESS` action.

But setting this under `FETCH_INIT` and `FETCH_SUCCESS` will still cause incorrect behavior: these actions are for the user details and are unrelated to the Points functionality so they should not modify any of the Points state. For example, if you tap "Fetch points" and then deny the points permission, you will correctly get the error state for Points. But currently if you were then to tap "Fetch user details" then it would reset the points error state.

Reply:
@corycaywood Yeah, as we discussed and you got the root cause, it had issues with fetching all the permissions by https://github.com/rakutentech/js-miniapp/blob/master/js-miniapp-sample/src/services/permissions/reducers.js 
Thank you for your suggestion 🙇